src/components/Dashboard/TodoList.vue

```diff
@@ -16,14 +16,14 @@
       :dialog="deleteErrorDialog"
       :errors="deleteErrors"
       :warnings="deleteWarnings"
-      @okay="resetErrors"
+      @okay="resetErrors()"
```

Comment:
FYI, the previous syntax works because it automatically passes all event parameters to the function, but I prefer this syntax as it makes it clearer that a function is called.